recipes/simage/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "simage" %}
+{% set version = "1.7.0" %}
+{% set sha256 = "4b57ed3c4997fbcc8b13823a60c1115cd97f701ada8afd44241f8512a5a9adc0" %}
+{% set download_url = "https://github.com/FreeCAD/pivy/releases/download/0.6.3/simage.tar.gz" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}.tar.gz
+    url: {{ download_url }}
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    skip: true  # [py<35]
+    skip: true  # [osx]
+
+    features:
+        - vc14  # [win]
+
+    track_features:
+        - vc14  # [win]
+
+requirements:
+    build:
+        - toolchain
+        - vc 14  # [win]
+        - ninja  # [win]
+        - gcc  # [linux]
+        - cmake
+        - qt 5.*
+        - libpng
+        - jpeg
```

Comment:
please pin libpng and jpeg as well according to this table: https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L64-L129